src/main/java/teammates/logic/api/GateKeeper.java

```diff
@@ -255,7 +248,7 @@ public void verifyAccessible(InstructorAttributes instructor, FeedbackSessionAtt
      * privilegeName for feedbackSession.
      */
     public void verifyAccessible(InstructorAttributes instructor, FeedbackSessionAttributes feedbacksession,
-                                 boolean creatorOnly, String privilegeName) {
```

Comment:
You should create your branch from the `master`. There should not be other's commit here.

Reply:
Forgot to merge. Sorry

Reply:
The change still exist?